NoSoft-SA/nspack
# Feature/robust handling of missing rmt container type

Summary of this change:
With the aim of easing new installations, if a default container type has not yet been created, an error will be raised as an reminder to do so during various rebin related operations.


CHANGELOG entry:

### Added
- `failed response` to [`convert_carton_to_rebin`](0f88d1e1df7bb895a0239acc4788ead36930dcf3)  and
- `Crossbeams::InfoError` to [rebin related UI rules](3cdd5a29210bab3a2c623eddb50ced9e2e37149d)

How to test this code via the UI:
If a rmt_container_type = client setting `DEFAULT_RMT_CONTAINER_TYPE` does not exist, the following should each produce an error. Therefore each should function as normal given the masterfile and client setting is set up correctly.

- | raw materials | deliveries | new/edit | scan bin groups/create bin groups/new rmt bin (dependent on `web bin scan mode`-field) |
- | raw materials | deliveries | new bin |
- | rmd | create/edit rebin | 
- | production | runs | (via action menu) create rebin |
- | production | runs | tip bin | (convert a carton to rebin through tipping it, see client rule `convert_carton_to_rebins` )
